python/apps/taiga/src/taiga/base/sampledata/test_data.py

```diff
@@ -64,7 +64,8 @@ async def load_test_data() -> None:
     for workspace in workspaces:
         # create one project (kanban) in each workspace with the same creator
         # it applies a template and creates also admin and general roles
-        project = await factories.create_project(workspace=workspace, created_by=workspace.created_by)
+        if workspace.created_by:
```

Comment:
What cases are those ones we don't have a created_by defined? Is it just to prevent a potential error while loading the data?

Reply:
Yes, it is just to prevent a error while loading data.